en/editor/project/index.md

```diff
@@ -1,76 +1,116 @@
+- Layers can be customized from 0 to 19, and clearing the input box removes the original settings.
+- The last 12 Layers are built-in to the engine and cannot be modified.
+- The locations where Layers are currently used include:
+
+  1. The `Layer` property of the node `Node` in the **Inspector** panel, For more details, please refer to the [Node Component](../../concepts/scene/node-component.md#setting-the-visibility-of-nodes) documentation.
+
+      ![Layers-node](./index/layers-node.png)
+
+  2. The `Visibility` property of the Camera node in the **Inspector** panel, the `Layer` property of the node matches the `Visibility` property of the Camera. A node can only be seen by the Camera if the `Layer` property set by the node is checked in the `Visibility` of the Camera. For more details, please refer to the [Camera Component](./../components/camera-component.md#camera-component) documentation.
+
+      ![Layers-camera](./index/layers-camera.png)
+
+<!-- The native engine settings changes mainly affect the use of cocos2dx engine templates when building native projects, and the changes can take effect in real time. -->
+
+## Physics
+
+![physics](./index/physics.png)
 
-## Texture Compress
+Used to configure various parameters of physics, please refer to the [Physics Config](physics-configs.md) documentation for details.
 
-> In Cocos Creator 3.0, the editor has modified the use of compressed texture configuration to configure presets in project settings and select presets for image asset's inspector. After the old version of the project is upgraded, the editor will automatically scan all the compressed texture configurations in the project and sort out as several presets.
+## Scripting
 
-Used to add compressed texture preset configuration, you can directly select the compressed texture preset to quickly add in the inspector of image asset. At the same time, after adding presets, you can also directly modify the presets to update batch texture compress configuration. Project settings allow users to add multiple compressed texture configurations, and each compressed texture configuration allows to add different format for different platform categories.
+![scripting](./index/scripting.png)
 
-Platform is rough devised as following:
+## Texture Compression
 
-1. Web: refers to the two platforms Web-Mobile and Web-Desktop
-2. Mac & Windows
-3. iOS
-4. Mini Game: Refers to the mini-games of various manufacturers' platforms, such as WeChat Mini Games and Huawei Quick Game Waiting;
-5. Android
+In Cocos Creator 3.0, the preset configuration of texture compression is moved to **Project Settings**, and developers can select how the image asset is preset in the **Inspector** panel. After upgrading to v3.0, the editor will automatically scan all the texture compression configurations in the project and sort out a few presets. Since it is automatically scanned, the preset names may not match the project and can be modified by developers.
```

Comment:
```suggestion
The preset configuration of texture compression is moved to **Project Settings**, and developers can select how the image asset is preset in the **Inspector** panel. The editor will automatically scan all the texture compression configurations in the project and sort out a few presets. Since it is automatically scanned, the preset names may not match the project and can be modified by developers.
```